indexclient/client.py

```diff
@@ -31,6 +31,20 @@ def handle_error(resp):
             resp.raise_for_status()
 
 
+def indexd_request_wrapper(func):
+    def retry_logic_with_timeout(*args, **kwargs):
+        kwargs.update({"timeout": 60})
```

Comment:
did you mean to use `set_default()`? The `update` here overwrites any given `timeout`.

Reply:
Good catch. Thanks!